plugin

```diff
@@ -273,6 +273,7 @@
   "silentbil/silent-remotes-card",
   "slipx06/sunsynk-power-flow-card",
   "sopelj/lovelace-kanji-clock-card",
+  "SpangleLabs/history-explorer-card",
```

Comment:
So you forked this to abandon it?
It looks like you stopped maintaining your fork, is that correct?

Reply:
I've been struggling to find time or motivation for it, must admit. Everything home assistant was seeming so unstable and unreliable and I gave up trying to keep it all working, sooner than I had hoped. :(

Reply:
Do you think you will get back to it, or should we close this one?

Reply:
Yes, I do think so. I had a very hectic last few months for life reasons. But I've been digging into HA card automated testing, (though, [having some issues on how to actually setup historic entity state](https://github.com/rianadon/hass-taste-test/issues/4) in a HA instance for that), and I've ordered a new zigbee dongle. Gotta give it another go. And definitely this card is essential for my use-cases.

So yes, hoping I can get testing set up and clean up the issues there, thanks